docs/commands/verify.md

```diff
@@ -23,7 +23,7 @@ If not, it tries to give you a hint that maybe you forgot to run `dtm init` firs
 
 ## 4 Config / State / Resource
 
-For definitions of _Config_, _State_, and _Resource_, see [Core Concepts](../core-concepts/core-concepts).
+For definitions of _Config_, _State_, and _Resource_, see [Core Concepts](../core-concepts/core-concepts.md).
```

Comment:
`.md` is no needed

Reply:
see here, it doesn't end with `.md` or `/`, but [the coresponding link](https://docs.devstream.io/en/latest/commands/core-concepts/core-concepts) is fail to render.
https://github.com/devstream-io/devstream/blob/90d0a07ab57b34d60ed8e8c59bf685f2d93d99b9/docs/commands/verify.md?plain=1#L26

So, I think I have to build docs with `mkdocs` in local. But I get the error `Config file 'mkdocs.yml' does not exist.`.

So, how should we build docs of dtm? I need your help, @daniel-hutao , thanks. And maybe we talk about it tommorrow, just go to hava a break now.

Reply:
![image](https://user-images.githubusercontent.com/18692628/172189044-e3c005cc-4c58-48c1-abe3-e0315b7aea66.png)

![image](https://user-images.githubusercontent.com/18692628/172189089-4ced8a66-340e-441d-99b7-048202c4a56a.png)

`mkdocs` can be used locally. Please ensure that the execution path is in the project root directory. If there are still errors, please let me know again.

Reply:
Sorry, I entered wrong path(`/docs`). Thank you. I will retry tomorrow, Good Night!